config.py

```diff
@@ -72,12 +72,19 @@ def _merge_options(base, rewrites):
     return result
 
 
-def _update_context_recursively(context):
+def _update_context_recursively(context, depth=0):
+    if depth > settings.MAX_CONFIG_INCLUDE_DEPTH:
```

Comment:
Why we need this limit?

Reply:
to protect against infinite recursion

Reply:
How we get infinite recursion here? And why default limit is 5?

Reply:
example:
config includes file A,
file A includes file B,
file B includes file A.

5 - empirical choice

Reply:
Ok, we can create table with filename: filecontent_hash and detect this situation w/o any limits. Just check file hash before read next file.

Reply:
:point_up: ef59c5bd360c04f93b640183a4a0174e0635b75e